mmv1/products/workstations/WorkstationConfig.yaml

```diff
@@ -369,6 +369,50 @@ properties:
               description: |
                 Name of the snapshot to use as the source for the disk. This can be the snapshot's `self_link`, `id`, or a string in the format of `projects/{project}/global/snapshots/{snapshot}`. If set, `sizeGb` and `fsType` must be empty. Can only be updated if it has an existing value.
               # TODO(esu): Add conflicting fields once complex lists are supported.
+  - !ruby/object:Api::Type::Array
+    name: 'ephemeralDirectories'
+    description: |
+      Ephemeral directories which won't persist across workstation sessions.
+    default_from_api: true
+    item_type: !ruby/object:Api::Type::NestedObject
+      properties:
+        - !ruby/object:Api::Type::String
+          name: 'mountPath'
+          description: |
+            Location of this directory in the running workstation.
+          default_from_api: true
+          immutable: true
```

Comment:
If these fields are `default_from_api`, is `immutable` still required? This will cause resource recreation if the value of the field changes.

Reply:
Removed